src/Migration/Destinations/Appwrite.php

```diff
@@ -439,6 +439,9 @@ protected function createColumn(Column $resource): bool
             Column::TYPE_IP,
             Column::TYPE_EMAIL,
             Column::TYPE_URL,
+            Column::TYPE_POINT => UtopiaDatabase::VAR_POINT,
+            Column::TYPE_LINE => UtopiaDatabase::VAR_LINESTRING,
+            Column::TYPE_POLYGON => UtopiaDatabase::VAR_POLYGON,
             Column::TYPE_ENUM => UtopiaDatabase::VAR_STRING,
```

Comment:
These need to be defined after the string cases, otherwise string, ip, email, url will now be treated as a point